apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -165,10 +173,11 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
             customerUniquename: this.invoiceFormDetails.voucherDetails.customerUniquename,
             voucherDate: this.invoiceFormDetails.voucherDetails.voucherDate,
             totalTaxableValue: Number(this.invoiceFormDetails.voucherDetails.totalTaxableValue),
-            subTotal: Number(this.invoiceFormDetails.voucherDetails.subTotal)
-
+            subTotal: Number(this.invoiceFormDetails.voucherDetails.subTotal),
+            tcsTotal: Number(this.invoiceFormDetails.voucherDetails.tcsTotal),
+            tdsTotal: Number(this.invoiceFormDetails.voucherDetails.tdsTotal)
         });
-        if (this.adjustPayment.grandTotal - this.adjustPayment.totalAdjustedAmount > 0) {
+        if (this.adjustPayment.grandTotal - this.adjustPayment.totalAdjustedAmount - this.depositAmount + this.adjustPayment.tdsTotal + this.adjustPayment.tcsTotal > 0) {
```

Comment:
Consider moving this expression into a typescript getter that will help to reduce the expression size and make it more readable